src/ruby/yast/wfm.rb

```diff
@@ -194,9 +194,17 @@ def self.ask_to_run_debugger?
     # @param [Exception] e the caught exception
     # @return [String] human readable exception description
     def self.internal_error_msg(e)
-      "Internal error. Please report a bug report with logs.\n" \
-        "Details: #{e.message}\n" \
-        "Caller:  #{e.backtrace.first}"
+      msg = "Internal error. Please report a bug report with logs.\n"
+
+      if e.is_a?(ArgumentError) && e.message =~ /invalid byte sequence in UTF-8/
+        msg += "A string was encountered that is not valid in UTF-8.\n" \
```

Comment:
In the end it still is an internal error and we still want a bug report with logs, so it is better to keep it.